modules/simple_forms_api/spec/services/pdf_filler_spec.rb

```diff
@@ -76,8 +89,12 @@
     end
 
     def read_form_mapping(form_number)
```

Comment:
I'm not convinced this test was ever properly populating the values for this JSON string. It definitely does now.

Reply:
ooo good catch!